changelog.d/20241211_144107_PL-133226-fix-changelog.sh_scriv.md

```diff
@@ -0,0 +1,16 @@
+<!--
+
+A new changelog entry.
+
+Delete placeholder items that do not apply. Empty sections will be removed
+automatically during release.
+
+Leave the XX.XX as is: this is a placeholder and will be automatically filled
+correctly during the release and helps when backporting over multiple platform
+branches.
+
+-->
+
+### NixOS XX.XX platform
+
+- Internal: Fix changelog.sh script (PL-133226)
```

Comment:
This doesn't require a changelog entry. Also, the changelog is a typical non-helpful entry, which is also an indicator of: just don't do it, as it's basically noise for the reader.